README.md

```diff
@@ -46,9 +46,24 @@ We refer to our [official benchmarks](https://www.kymat.io/userguide.html#benchm
 
 ## How to cite
 
-If you use this package, please cite the following paper:
-
-Andreux M., Angles T., Exarchakis G., Leonarduzzi R., Rochette G., Thiry L., Zarka J., Mallat S., Andén J., Belilovsky E., Bruna J., Lostanlen V., Hirn M. J., Oyallon E., Zhang S., Cella C., Eickenberg M. (2019). Kymatio: Scattering Transforms in Python. arXiv preprint arXiv:1812.11214. [(paper)](https://arxiv.org/abs/1812.11214)
+If you use this package, please cite our paper <a href="">Kymatio: Scattering Transforms in Python</a>:
```

Comment:
Can we keep the traditional citation and include a link to the BibTeX entry? Looks quite ugly (and not all of us are fluent in BibTeX).